docs/content/querying/granularities.md

```diff
@@ -11,11 +11,168 @@ It can be specified either as a string for simple granularities or as an object
 
+#### Example:
+
+Suppose you have data below stored in Druid,
+
+``` json
+{"timestamp": "2013-08-31T01:02:33Z", "page": "AAA", "language" : "en"}
+{"timestamp": "2013-09-01T01:02:33Z", "page": "BBB", "language" : "en"}
+{"timestamp": "2013-09-02T23:32:45Z", "page": "CCC", "language" : "en"}
+{"timestamp": "2013-09-03T03:32:45Z", "page": "DDD", "language" : "en"}
+```
+
+After submitting a groupBy query with `hour` granularity,
+
+``` json
+{
+   "queryType":"groupBy",
+   "dataSource":"my_dataSource",
+   "granularity":"hour",
+   "dimensions":[
+      "language"
+   ],
+   "aggregations":[
+      {
+         "type":"count",
+         "name":"count"
+      }
+   ],
+   "intervals":[
+      "2000-01-01T00:00Z/3000-01-01T00:00Z"
+   ]
+}
+```
+
+you will get
+
+``` json
+[ {
+  "version" : "v1",
+  "timestamp" : "2013-08-31T01:00:00.000Z",
+  "event" : {
+    "count" : 1,
+    "language" : "en"
+  }
+}, {
+  "version" : "v1",
+  "timestamp" : "2013-09-01T01:00:00.000Z",
+  "event" : {
+    "count" : 1,
+    "language" : "en"
+  }
+}, {
+  "version" : "v1",
+  "timestamp" : "2013-09-02T23:00:00.000Z",
+  "event" : {
+    "count" : 1,
+    "language" : "en"
+  }
+}, {
+  "version" : "v1",
+  "timestamp" : "2013-09-03T03:00:00.000Z",
+  "event" : {
+    "count" : 1,
+    "language" : "en"
+  }
+} ]
+```
+
+Note that all the empty buckets are discarded.
+
+
+If you change the granularity to `day`, you will get
+
+``` json
+[ {
+  "version" : "v1",
+  "timestamp" : "2013-08-31T00:00:00.000Z",
+  "event" : {
+    "count" : 1,
+    "language" : "en"
+  }
+}, {
+  "version" : "v1",
+  "timestamp" : "2013-09-01T00:00:00.000Z",
+  "event" : {
+    "count" : 1,
+    "language" : "en"
+  }
+}, {
+  "version" : "v1",
+  "timestamp" : "2013-09-02T00:00:00.000Z",
+  "event" : {
+    "count" : 1,
+    "language" : "en"
+  }
+}, {
+  "version" : "v1",
+  "timestamp" : "2013-09-03T00:00:00.000Z",
+  "event" : {
+    "count" : 1,
+    "language" : "en"
+  }
+} ]
+```
+
+
+If you change the granularity to `none`, you will get
+
+``` json
+[ {
+  "version" : "v1",
+  "timestamp" : "2013-08-31T01:02:33.000Z",
+  "event" : {
+    "count" : 1,
+    "language" : "en"
+  }
+}, {
+  "version" : "v1",
+  "timestamp" : "2013-09-01T01:02:33.000Z",
+  "event" : {
+    "count" : 1,
+    "language" : "en"
+  }
+}, {
+  "version" : "v1",
+  "timestamp" : "2013-09-02T23:32:45.000Z",
+  "event" : {
+    "count" : 1,
+    "language" : "en"
+  }
+}, {
+  "version" : "v1",
+  "timestamp" : "2013-09-03T03:32:45.000Z",
+  "event" : {
+    "count" : 1,
+    "language" : "en"
+  }
+} ]
+```
+
+Note that although `none` means millisecond granularity, if the aggregation granularity you specified during ingestion is bigger than millisecond,
+then `none` is essentially your aggregation granularity. See `queryGranularity` in [Ingestion Spec](../ingestion/index.html).
```

Comment:
I believe it will be more clear if we said, "at ingestion time `none` means millisecond granularity and at query time `none` means the granularity used at ingestion time."